pkg/util/env_resolver.go

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
I know that I'm just using one of the helpers, but I copied/pasted them from KEDA and having them here can help in the future (or not ❓ )